.github/workflows/deploy-stage.yml

```diff
@@ -0,0 +1,133 @@
+name: Staging Deploy
+
```

Comment:
One thing we should include here in either a README or a doc block are all the secrets you'll need to set. It can be challenging to scan these files and figure out what needs set.

Reply:
Added docs.